draft-ietf-rats-eat.md

```diff
@@ -1827,28 +1832,58 @@ The JWT Claim Names and CWT Claim Keys are not expected to change.
 * Claim Name: UEID
 * Claim Description: The Universal Entity ID
 * JWT Claim Name: "ueid"
-* CWT Claim Key: 11
+* CWT Claim Key: 256
```

Comment:
Please update all the "CWT Claim Key" values to use the form:

- CWT Claim Key: TBD (requested value NNN)

For instance, this example should be:

- CWT Claim Key: TBD (requested value 256)